### PR TITLE
Add RemarkCommand

### PR DESCRIPTION
**Pull request by Zxun2**
Taken from the compulsory tutorial submission. Seems like a pretty useful command to have.

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 1]
```diff
@@ -25,10 +25,10 @@ public class TypicalPersons {
 
     public static final Person ALICE = new PersonBuilder().withName("Alice Pauline")
             .withAddress("123, Jurong West Ave 6, #08-111").withEmail("alice@example.com")
-            .withPhone("94351253")
+            .withPhone("94351253").withRemark("She likes aardvarks.")
             .withTags("friends").build();
     public static final Person BENSON = new PersonBuilder().withName("Benson Meier")
-            .withAddress("311, Clementi Ave 2, #02-25")
+            .withAddress("311, Clementi Ave 2, #02-25").withRemark("He can't take beer!")
```
Comment (gohyongjing) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:31]: should withRemark be on a new line similar to the other with() methods?

```java
.withAddress("311, Clementi Ave 2, #02-25")
.withRemark("He can't take beer!")
```

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -75,7 +77,7 @@ public class CommandTestUtil {
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```
Comment (gohyongjing) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:80]: Should the indentation stick to 8 spaces?

**Review by gohyongjing — COMMENTED** [review]
LGTM!

**Comment by Zxun2** [comment]
@gohyongjing thanks!

**Pull request closed without merging**